frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-loader.ts

```diff
@@ -105,6 +107,27 @@ export const registerPluginEntryCallback = (pluginStore: PluginStore) => {
   );
 };
 
+export const loadPluginFromURL = async (baseURL: string) => {
+  const manifest = await fetchPluginManifest(baseURL);
+  return loadDynamicPlugin(baseURL, manifest);
```

Comment:
FYI, there was `return await` here but I've removed the `await` to simply return the promise itself.

Reference: https://eslint.org/docs/rules/no-return-await